services/basic_types.proto

```diff
@@ -1333,6 +1348,11 @@ message ServiceEndpoint {
      * The port of the node
      */
     int32 port = 2;
+
+    /**
+     * the fully qualified domain name of the node
+     */
+    string fqdn = 3;
```

Comment:
May be `domain_name` instead of `fqdn` ?

Reply:
Updated.